### ISSUE TITLE
Skipped items only excludes first item

### ISSUE THREAD

**Issue by bob-van-der-bibs** [bdkjones/CodeKit-1-Old#190]
It seems that the skipped items functionality doesn't fully work.
i ecluded a few folders by adding they're name to the list

but only the first 1 is actually excluded the others just show up in the all files tab

RDG

**Comment by bdkjones** [comment]
Can you be more specific? What entries are in the skipped items list? What items are showing up in the all files tab that should have been skipped?

I need full-path examples.

**Comment by bob-van-der-bibs** [comment]
hi 

We just figured it out.

We made the project and excluded the 'core' folder. all went well
the we added the 'assets' and 'manager' folders to the skip list.
nothing happened
we refreshed but nothing happened,. We expected to rescan on adding a folder to the skip list but it doesn't do that.
we removed the whole project, and the re added it .
the the excluded files were ok.

might be nice to rescan on addFolderToSkipList

PS might be nice to have a skip list per project, 

Bob
